pixl_ehr/src/pixl_ehr/main.py

```diff
@@ -58,6 +62,21 @@ async def startup_event() -> None:
         task.add_done_callback(background_tasks.discard)
 
 
+@app.post(
+    "/export-radiology-as-parquet",
+    summary="Copy all radiology reports in the PIXL DB to a parquet file",
+)
+def export_radiology_as_parquet(project_name: str, extract_datetime: datetime) -> Path:
+    """Batch export of all radiology reports in PIXL DB to a parquet file."""
+    # This command has to tell us the extract identifiers so we can extract to the correct
+    # path, however we can't verify that the reports in the DB actually came from that extract.
+    # Also we can't check that all reports in the queue have been processed.
+    # So we are relying on the user passing correct parameters here.
+    anon_data = PIXLDatabase().get_radiology_reports()
+    pe = ParquetExport(project_name, extract_datetime)
+    return pe.export_radiology(anon_data)
```

Comment:
```
error: Returning Any from function declared to return "Path"  [no-any-return]
```
Why this error when `pe.export_radiology` *is* annotated to return `Path`?

Reply:
Yeah I'm not sure about that one, if you look for `export_radiology` in the `.mypy_cache` then it does have the return type as pathlib.Path

```
"export_radiology": {
            ".class": "SymbolTableNode",
            "kind": "Mdef",
            "node": {
              ".class": "FuncDef",
              "abstract_status": 0,
              "arg_kinds": [
                0,
                0
              ],
              "arg_names": [
                "self",
                "anon_data"
              ],
              "dataclass_transform_spec": null,
              "flags": [],
              "fullname": "pixl_core.src.core.omop.ParquetExport.export_radiology",
              "name": "export_radiology",
              ...
                "ret_type": "pathlib.Path",
                ...
```

Might just need to manually set it because clearly something in mypy isn't quite right

```suggestion
    parquet_file: Path = pe.export_radiology(anon_data)
    return parquet_file
```

Reply:
👌 the API wraps around a method which returns the value but the API method doesn't?